### ISSUE TITLE
reclass.cells function throwing an error

### ISSUE THREAD

**Issue by adiamb** [chris-mcginnis-ucsf/MULTI-seq#16]
Hey chris, when I try to run the  
 `reclass.cells <- findReclassCells(barTable.full, neg.cells = names(final.calls)[which(final.calls=="Negative")])`.   
it throws an error 
`[1] "Normalizing barode data..."
[1] "Pre-allocating data structures..."
Error in `.rowNamesDF<-`(x, value = value) : 
  duplicate 'row.names' are not allowed
In addition: Warning message:
non-unique values when setting 'row.names': `.  
what am i Missing, if pass a unique command to the neg.cell argument, then it works but then I find duplicated cell.ids where one instance is negative while other is attributed to a sample barcode.

**Comment by adiamb** [comment]
I wrote a companion python preprocessing which is faster than using R to handle fastq processing, I wonder if you would like to test it out give me your feedback? https://github.com/Mignot-Lab/Multiseq10x

**Comment by chris-mcginnis-ucsf** [comment]
Hey @adiamb ,

I've seen this error before but unfortunately do not exactly remember the reason. I sort of remember this happening if I tried to move on to semi-supervised reclassification before I ran the classification steps until there were no negatives... but I could be wrong. Did you ever figure this out?

And those new python scripts look impressive! I'd consider incorporating if I (or you?) could interface it with R using the reticulate package. I'd like to try to keep the necessary-experience floor for using deMULTIplex low by not requiring users to be versed in both R and python. Feel free to reach out to me at chris.mcginnis@ucsf.edu if you want to chat more.

Chris
